app/views/admin/tenants/index.html.erb

```diff
@@ -1,69 +1,81 @@
-<div id="tenants_header">
-  <h1>Borrowers</h1>
+<div class ="col-md-12">
+  <div id="tenants_header">
+    <h1>Borrowers</h1>
+  </div>
 </div>
-<div id="unapproved">
-  <h2>Unapproved Borrowers</h2>
-  <table style="width:100%">
-    <tr>
-      <th>Organization</th>
-      <th>Location</th>
-    </tr>
+<div class="container">
+  <p class="lead">Unapproved Borrowers</p>
 
-    <% if @tenants[0] %>
-      <% @tenants[0].each do |tenant| %>
-        <tr>
-          <td><%= link_to tenant.organization,
-            tenant_dashboard_path(slug: tenant.slug) %></td>
-          <td><%= tenant.location %></td>
-          <td><%= link_to "EDIT", edit_admin_tenant_path(tenant.id) %></td>
-        </tr>
+  <div class="col-md-12">
+    <table class="table table-bordered table-hover">
+      <th class="text-left">Organization</th>
+      <th class="text-left">Location</th>
+      <th class="text-center">Control</th>
+      <% if @tenants[0] %>
+        <% @tenants[0].each do |tenant| %>
+          <tr>
+            <td>
+              <%= link_to tenant.organization, tenant_dashboard_path(slug: tenant.slug) %>
+            </td>
+            <td>
+              <%= tenant.location %>
+            </td>
+            <td>
+              <%= button_to "Edit", edit_admin_tenant_path(tenant.id), class: 'btn btn-info btn-xs btn-block', method: :get %>
+            </td>
+          </tr>
+        <% end %>
       <% end %>
-    <% else %>
-      None
-    <% end %>
-  </table>
-</div>
-<div id="inactive">
-  <h2>Inactive Borrowers</h2>
-  <table style="width:100%">
-    <tr>
-      <th>Organization</th>
-      <th>Location</th>
-    </tr>
+    </table>
+  </div>
+
+  <p class="lead">Inactive Borrowers</p>
 
-    <% if @tenants[1] %>
-      <% @tenants[1].each do |tenant| %>
-        <tr>
-          <td><%= link_to tenant.organization,
-            tenant_dashboard_path(slug: tenant.slug) %></td>
-          <td><%= tenant.location %></td>
-          <td><%= link_to "EDIT", edit_admin_tenant_path(tenant.id) %></td>
-        </tr>
+  <div class="col-md-12">
+    <table class="table table-bordered table-hover">
+      <th class="text-left">Organization</th>
+      <th class="text-left">Location</th>
+      <th class="text-center">Control</th>
+      <% if @tenants[1] %>
+        <% @tenants[1].each do |tenant| %>
+          <tr>
+            <td>
+              <%= link_to tenant.organization, tenant_dashboard_path(slug: tenant.slug) %>
+            </td>
+            <td>
+              <%= tenant.location %>
+            </td>
+            <td>
+              <%= button_to "Edit", edit_admin_tenant_path(tenant.id), class: 'btn btn-info btn-xs btn-block', method: :get %>
+            </td>
+          </tr>
+        <% end %>
       <% end %>
-    <% else %>
-      None
```

Comment:
The way it is structured right now, it is no longer necessary. I think its better without.